common/component/sqlserver/drop_schema.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sqlserver
+
+import (
+	"context"
+	"database/sql"
+)
+
+// DropSchema drops a schema from a SQL Server database, including all resources that were created inside
+// Adapted from: https://stackoverflow.com/a/76742742/192024
+func DropSchema(ctx context.Context, db *sql.DB, schema string) error {
```

Comment:
Reviewers: this utility is currently only used in the  test for migrations, but added here as it can be helpful for other tests in the future too (e.g. conformance tests for the state store)